pkg/image/origins.go

```diff
@@ -178,6 +178,8 @@ var OriginMap = map[string]string{
 	"mirrored-neuvector-manager":                              "https://github.com/neuvector/manager",
 	"mirrored-neuvector-scanner":                              "https://github.com/neuvector/scanner",
 	"mirrored-neuvector-updater":                              "https://github.com/neuvector/neuvector",
+	"mirrored-neuvector-prometheus-exporter":                  "https://github.com/neuvector/prometheus-exporter",
+	"mirrored-neuvector-registry-adapter":                     "https://github.com/neuvector/registry-adapter",
```

Comment:
According to the logs the missing images are `neuvector-csp-adapter` and `registry-adapter`.

Can you update the origins accordingly?

```
2023/07/17 18:33:05 [ERROR] not all images have a source code origin defined. Please provide origin URL's within rancher/pkg/image/origins.go for the following images: 
neuvector-csp-adapter
registry-adapter
```